x-pack/plugins/security_solution/public/flyout/right/components/response_button.tsx

```diff
@@ -45,31 +34,22 @@ export const ResponseButton: React.FC = () => {
 
   return (
     <>
-      {!responseActions ? (
-        <p data-test-subj={RESPONSE_EMPTY_TEST_ID}>
-          <FormattedMessage
-            id="xpack.securitySolution.flyout.right.response.noDataDescription"
-            defaultMessage="There are no response actions defined for this event."
-          />
-        </p>
-      ) : (
-        <EuiButton
-          onClick={goToResponseTab}
-          iconType="documentation"
-          data-test-subj={RESPONSE_BUTTON_TEST_ID}
-          aria-label={i18n.translate(
-            'xpack.securitySolution.flyout.right.response.responseButtonAriaLabel',
-            {
-              defaultMessage: 'Response',
-            }
-          )}
-        >
-          <FormattedMessage
-            id="xpack.securitySolution.flyout.right.response.responseButtonLabel"
-            defaultMessage="Response"
-          />
-        </EuiButton>
-      )}
+      <EuiButton
+        onClick={goToResponseTab}
+        iconType="documentation"
+        data-test-subj={RESPONSE_BUTTON_TEST_ID}
+        aria-label={i18n.translate(
+          'xpack.securitySolution.flyout.right.response.responseButtonAriaLabel',
+          {
+            defaultMessage: 'Response',
+          }
+        )}
+      >
+        <FormattedMessage
+          id="xpack.securitySolution.flyout.right.response.responseButtonLabel"
+          defaultMessage="Response"
+        />
```

Comment:
Since the aria label and the button's text content are identical, there is no need for the label here.

```suggestion
      >
        <FormattedMessage
          id="xpack.securitySolution.flyout.right.response.responseButtonLabel"
          defaultMessage="Response"
        />
```

You might need to make sure to remove `xpack.securitySolution.flyout.right.response.responseButtonAriaLabel` from the translations as well.

Reply:
Thanks, removed the aria-label :) There are no translations for this yet 👍